bridge/bridge.js

```diff
@@ -0,0 +1,44 @@
+let MiniAppBridge = {}
```

Comment:
This will be called from the native side, so we need it to be scoped globally (`let` is for local scoping). This isn't going to be accessible to the calls from the native side (at least not for Android).

If you're worried about polluting the JS side with more objects, you could set it on the `MiniApp` object. i.e.
```
window.MiniApp = {};
MiniApp.bridge = MiniAppBridge;
MiniApp.getUniqueId = ....
```

Reply:
Sorry, I had second thoughts on this. See my comment below